### DISCUSSION TITLE
[Feature]Swipe Controls

### DISCUSSION THREAD

**Discussion by PiyushBaria** [libre-tube/LibreTube#854] [Ideas]
Display brightness and volume control by swipe

**Comment by Donkey-Doug** [comment]
Hi, maybe take a look at vlc for android. It has

1. Horizontal scrolling to scrub
2. Vertical on right half for brightness
3. Vertical in left half for audio

**Comment by Schninzent** [comment]
Swipe Control would be so neat! This would make LibreTube perfect. After using Vanced and NewPipe I just can't live without it.

**Comment by Rorschach1010** [comment]
Same here. In 2022 a player needs swipe control. Android itself already totally works with swipe controls. Players profit from a way better visibility as you actually dont need any buttons at all anymore when 10sec skips, pause/play and volume is controlled via swipe.

**Comment by ghost** [comment]
This gesture has been applied in the recent debug build. Look #2040

**Comment by Rorschach1010** [comment]
That's awesome, big thanks to everyone involved. Libretube is (imo) now officially better then newpipe.

**Comment by ghost** [comment]
One can join the LibreTube Alpha Builds Channel ➡️ https://t.me/LibreTubeAlpha to try out all the new updates implemented, before each complete beta version is released.
